core/src/main/scala/org/http4s/parser/SimpleHeaders.scala

```diff
@@ -94,6 +94,12 @@ private[parser] trait SimpleHeaders {
     }
   }.parse
 
+  def AGE(value: String): ParseResult[Age] = new Http4sHeaderParser[Age](value) {
+    def entry = rule {
+      Digits ~ EOL ~> ((t: String) => Age.unsafeFromDuration(FiniteDuration(t.toLong, SECONDS)))
```

Comment:
I'm not sure about this usage either. We know it is a positive so it should be ok, but still